### DISCUSSION TITLE
otlpreceiver missing on opentelemetry-collector-contrib

### DISCUSSION THREAD

**Discussion by sanjay-kamani-swi** [open-telemetry/opentelemetry-collector-contrib#15713] [Q&A]
is otlpreceiver missing on opentelemetry-collector-contrib? i see otlpjsonfilereceiver but not the otlpreceiver. 
The opentelemetry-collector repo has the otlpreceiver but the 'contrib' is missing the one, I think.

Any idea what's the reason? I have nginx instrumented with telemetry traces which i want to receive to collector-contrib and then export to another otlp endpoint. I could do this with opentelemetry-collector but it doesn't have other receiver/exporter that the contrib has, so i need to use the contrib for this purpose. any suggestion/alternative?

Thanks!

**Comment by chennin** [comment]
The `contrib` flavor includes the base flavor. You can confirm available receivers with the `components` subcommand.

```
Usage:
  otelcol-contrib [flags]
  otelcol-contrib [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  components  Outputs available components in this collector distribution
  help        Help about any command
```

`otlp` is there for me. (otelcol-contrib 0.69.0 here)
